src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -463,7 +486,7 @@ async function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourc
       }
 
       // -1 enables debugging with logging disabled. 0 disables debugging entirely.
-      MONO.mono_wasm_load_runtime(appBinDirName, hasDebuggingEnabled() ? -1 : 0);
+      MONO.mono_wasm_load_runtime('unused', hasDebuggingEnabled() ? -1 : 0);
```

Comment:
because we don't know who else besides Blazor is using it. 

Later we may deprecate whole `MONO` and `BINDING` things, like this https://github.com/dotnet/runtime/pull/79622 
but for now Blazor is not ready and will need few more PRs before we get there.